src/commands/transaction/mosaic.ts

```diff
@@ -31,31 +31,33 @@ import {
 import * as readlineSync from 'readline-sync';
 import {AnnounceTransactionsCommand, AnnounceTransactionsOptions} from '../../announce.transactions.command';
 import {OptionsResolver} from '../../options-resolver';
+import {NumericStringValidator} from '../../validators/numericString.validator';
 
 export class CommandOptions extends AnnounceTransactionsOptions {
     @option({
         flag: 'a',
         description: 'Initial supply of mosaics.',
+        validator: new NumericStringValidator(),
     })
-    amount: number;
+    amount: string;
 
     @option({
         flag: 't',
-        description: 'Mosaic transferable.',
+        description: '(Optional) Mosaic transferable.',
```

Comment:
Not 100% sure here. Should transferable and mutable be `Optional`? or just the restrictable  to be optional with default value = false?

Reply:
Example: The command takes the flag ``--transferable`` as optional. If you don't set it, this makes the mosaic non-transferable.